modules/install-sno-generate-the-discovery-iso-manually.adoc

```diff
@@ -0,0 +1,152 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso-manually_{context}"]
+= Generate the discovery ISO manually
+
+Installing {product-title} on a single node requires a discovery ISO, which you can generate with the following procedure.
+
+.Procedure
+
+. Download the {product-title} client (`oc`) and make it available for use:
```

Comment:
We have updated [guidance](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output) to "include an explicit instruction to 'run' or 'enter' the command." So these steps would look like something similar to: 

"Download the {product-title} client (`oc`) and make it available for use by entering the following command:" 

This may be outside of the scope of this PR since it would require updating additional modules, but I wanted to let you know.